### ISSUE TITLE
Layer: adding vectorised roads, railways and rivers for Britain & Loughborough

### ISSUE THREAD

**Issue by MKIM1008** [colouring-cities/colouring-britain#289]
@matkoniecz, could you add a new layer option in the layers drop-down saying "Roads, Rail & Rivers"? - with an option of ticking one or more of the 3 layers?

I will share the GIS file via email: 
- Motorways for Britain
- A-roads for Britain 
- B-roads for Britain
- C-roads for Loughborough and London
- Rail for Britain
- Rivers for Britain

Could you make the file size lighter for the layers on the website? - we don't need to interrogate any of these.

**Comment by MKIM1008** [comment]
Sample image 1_roads & rail:
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/dfb97b4a-1a22-4ac1-a29c-aadf3c56dfd0)

Sample image 2_rivers:
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/4c8b4538-3bab-4ca2-a89a-479e5af8effd)

**Comment by MKIM1008** [comment]
Road:
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/76c2d403-7b3d-431d-92dc-0255fd44ee04)
(C-road)
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/10a53dab-261a-4344-b53c-2aa282a330ed)

Rivers:
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/4cf8e2fa-c4f6-494b-8e47-74fe78eaa7c4)

**Comment by matkoniecz** [comment]
1) in case of just displaying OpenStreetMap data we need to credit them ( https://osmfoundation.org/wiki/Licence/Community_Guidelines/Horizontal_Map_Layers_-_Guideline )

2) limitation is setting up server to serve large amount of data (3 400 MB for roads, in sample that I got)

**Comment by MKIM1008** [comment]
https://github.com/colouring-cities/colouring-britain/issues/288

**Comment by MKIM1008** [comment]
Please review the boundary plan and add the layer button to the platform. 
*****************************
**Zoom level I (Britain countries)**
- Add country boundaries for England, Scotland and Wales 
 ![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/6dd107bb-7b49-4e14-b99f-77182fbef892)

**Zoom level II**
- Add regional boundaries for England with Scotland and Wales with names 
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/0482ad4f-a671-4df6-be9f-f220da3e31e9)

**Zoom level III**
- Add Built Up Areas for Britain
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/76604985-c548-4100-ace3-b9820d43106a)

**Zoom level IV**
- Zoom to Region level
- Add County boundaries for London and East Midlands without Town boundaries 
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/22ebb51e-0e1a-4bef-b8b5-e1aa5a500e40)

- (Don't include in the zoom but add only an option to the drop-down layers, and see the note below) Add Local Authority boundaries in East Midlands

**Zoom level V**
- OS Mastermap polygon for Loughborough
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/bed63f41-7da8-47c1-950d-1759d4540be7)
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/6ed94d39-54bb-4710-a57c-612d1eacde5e)

**In Layers**
- Under the administrative boundaries button, create the drop-down with the following layers: 
1. Regions
2. Built-Up areas
3. Counties
4. Local Authorities
****************************
Pleaes let us know if there's something that should be amended.

When it's confirmed and upon your request, I will share the Ordnance Survey/ONS boundary data.

**Comment by MKIM1008** [comment]
@matkoniecz, could you add two new layer options in the layers drop-down as below?
I will share the GIS file via email for you;

1) "Administrative boundaries" - with an option of ticking one or more of the 3 layers
- Country boundaries for Britain 
- Built Up Area polygons for Britain
- Regional boundaries for England
- County boundaries for England
- Local Authority boundaries for E.Midlands (already in place for London)

2) "Geographical areas for census statistics" 
- LSOAs for Loughborough 
- MSOAs for Loughborough 
- OAs for Loughborough

**Comment by MKIM1008** [comment]
@polly64

Should the census boundaries (LSOA, MSOA, OA) be limited in certain local boundary? (e.g. Charnwood or East Midlands)
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/3e7f4775-217a-4a03-b26b-e87693d5700e)

This is an image of LSOA from ONS. 
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/538b72e8-c2b5-4f87-b28d-42b7f5e373d7)

**Comment by MKIM1008** [comment]
@matkoniecz,

- [ ] Could you please add an additional layer called transportation network?
- [ ] Create a drop-down in the transportation layer with the following classes?
(i) Railways and rail stations
(ii) Rivers, ports and canals
(iii) Airports
(iv) Motorways 
(v) A-roads
(vi) B-roads

- [ ] Create a drop-down in the administrative boundaries layer with the following classes?
(i) Countries
(ii) Regions
(iii) Counties
(iv) Local Authorities

- [ ] Create a drop-down in the census / statistical geography boundaries layer with the following classes?
(i) Built-Up Areas
(ii) LSOAs
(iii) MSOAs
(iv) OAs

- [ ] Add Motorways @MKIM1008 will send

- [ ] Add LSOAs for Loughborough only @MKIM1008 will send

Could you prioritise this issue and do as much as possible before lunch tomorrow? Thank you!

**Comment by matkoniecz** [comment]
> Could you prioritise this issue and do as much as possible before lunch tomorrow? Thank you!

Yes. From what I see that would translate to "before 1 PM UK time", right?

**Comment by polly64** [comment]
@matkoniecz actually 12.30 would be ideal as I haver to leave for  first meeting at    12 but if you get anything  to sample before then do let me know so I can just test before I leave. The transport /admin boundary meeting at 12.30 Uk time. The land parcel meeting at 2pm Uk time. thanks!

**Comment by matkoniecz** [comment]
> Could you please add an additional layer called transportation network?

@MKIM1008 

Just to confirm.. That would be within these, right?

![screen02](https://github.com/colouring-cities/colouring-britain/assets/899988/78c5ff67-9a25-45f8-ad5c-99b4d0cde876)

**Comment by matkoniecz** [comment]
Also, is it possible to shorten "census / statistical geography boundaries" text a bit so it fits somehow?

**Comment by MKIM1008** [comment]
@matkoniecz,

Thank you for your time. I've sent 1) Motorway and 2) LSOAs for Loughborough-only for test, this morning and please let me know if you find any errors.

**Comment by matkoniecz** [comment]
Motorway layer is also up.

**Comment by matkoniecz** [comment]
In red, should be blue but too late for change. Borders are grouped in one and without example, hopefully it is not too bad.

(can add it but would require reloading server)

**Comment by matkoniecz** [comment]
![screen03](https://github.com/colouring-cities/colouring-britain/assets/899988/a23c68d2-7c61-4159-bf97-bf239c650fc4)

**Comment by matkoniecz** [comment]
note: forests and some areas seems to have no inspire parcels

**Comment by matkoniecz** [comment]
Uploaded INSPIRE parcels.

**Comment by matkoniecz** [comment]
INSPIRE parcel upload is working.

![screen08](https://github.com/colouring-cities/colouring-britain/assets/899988/6e1882fc-4a91-4160-9384-e865fa550746)

**Comment by polly64** [comment]
@matkoniecz wow. thanks so much for doing this in time

**Comment by matkoniecz** [comment]
Even nice would be doing it with a nice buffer :)

Hope that what I prepared was good enough for showing proof of concept.

**Comment by MKIM1008** [comment]
@matkoniecz,

This is the colour scheme Polly mentioned during the meeting today. 

![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/7eba8f8d-115f-4c24-9891-31070980eed1)

**Comment by MKIM1008** [comment]
@mdsimpson42,

I've sent census boundaries for Loughborough. I will send transport layers soon via email.

**Comment by MKIM1008** [comment]
- [ ] Airports
- [ ] Ports
- [ ] Canals
- [ ] Navigated rivers

**Comment by MKIM1008** [comment]
@polly64 
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/b7958384-79d3-4cde-b37e-54fa540dfa42)

Local Authority borders including city borders

**Comment by MKIM1008** [comment]
Coastal lines comparison

BFE from ONS ((BFE) Full resolution - extent of the realm (usually this is the Mean Low Water mark but, in some cases, boundaries extend beyond this to include offshore islands))
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/a4180f13-618d-4ea9-b98d-8e96bd0e79d9)

OS 
![image](https://github.com/colouring-cities/colouring-britain/assets/88775529/e79fefc6-73b7-4cc3-8b97-a0dc8ec8b4c8)

**Comment by MKIM1008** [comment]
Layer of logistics or freights: it's recommended to add and also may be relevant with a code in Land Use (but for economic analysis.)

https://findtransportdata.dft.gov.uk/browse?page=1
https://data.europa.eu/en/publications/datastories/open-transport-data-european-data-portal

**Comment by matkoniecz** [comment]
regions, counties and motorways are now available as layers
